<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Grafico</class>
 <widget class="QDialog" name="Grafico">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grafico</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::MailAttachment"/>
  </property>
  <widget class="QTableWidget" name="tbt_votos">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>256</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>128</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>CJ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Leon</string>
    </property>
   </column>
  </widget>
  <widget class="QProgressBar" name="pgs_cj">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>31</width>
     <height>111</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::Direction::TopToBottom</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="pgs_leon">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>31</width>
     <height>111</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_total">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>total de votos:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
